From a8292eb98d0694477752c159ec8aa21a767b194f Mon Sep 17 00:00:00 2001
From: Shashank Babu Chinta Venkata <sbchin@codeaurora.org>
Date: Mon, 9 Sep 2019 17:48:16 -0700
Subject: [PATCH] NOUPSTREAM: ANDROID: include: drm: increase DRM max property
 count to 64

Increase drm property count from 24 to 64 to make
room for custom properties of drm plane and drm
crtc objects.

[CPNOTE: 19/07/21] Lee: Pinged for status on the bug
[CPNOTE: 21/07/21] Lee: Code only supports vendor's features - no upstream user

Signed-off-by: Shashank Babu Chinta Venkata <sbchin@codeaurora.org>
Bug: 139653858
Change-Id: I3e42c1fc684d7d7cea828fc35b96673bdef598f4
Signed-off-by: Lee Jones <joneslee@google.com>
---
 include/drm/drm_mode_object.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/drm/drm_mode_object.h b/include/drm/drm_mode_object.h
index 912f1e415685..08d7a7f0188f 100644
--- a/include/drm/drm_mode_object.h
+++ b/include/drm/drm_mode_object.h
@@ -60,7 +60,7 @@ struct drm_mode_object {
 	void (*free_cb)(struct kref *kref);
 };
 
-#define DRM_OBJECT_MAX_PROPERTY 24
+#define DRM_OBJECT_MAX_PROPERTY 64
 /**
  * struct drm_object_properties - property tracking for &drm_mode_object
  */
-- 
2.34.1

